code/modules/mob/living/simple_animal/hostile/syndicate_mobs.dm

```diff
@@ -357,23 +353,23 @@
 	speed = 1.5
 	death_sound = 'sound/mecha/mechmove03.ogg'
 	loot = list(/obj/effect/mob_spawn/human/corpse/syndicatecommando,
```

Comment:
Since override loot in our initialize, I do not believe this would change anything and can be removed.

Reply:
The override only applies 50% of the time, so we still need this list for the other 50%